packages/storefront-lwc/src/modules/commerce/basket/basket.js

```diff
@@ -48,7 +48,7 @@ export default class Basket extends LightningElement {
     }
 
     get selectedShippingMethodId() {
-        return ShoppingBasket.basket.selectedShippingMethodId;
+        return this.basket.selectedShippingMethodId;
```

Comment:
clean up: just rely on this.basket as the singleton is going away (in Abhinav's work).